.github/workflows/firebase-client-hosting-merge.yml

```diff
@@ -16,7 +16,7 @@ env:
   NEXT_PUBLIC_BACKEND_BASE_URL: ${{secrets.VITE_BACKEND_BASE_URL}}
   NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY: ${{secrets.TEST_STRIPE_PK}}
   NEXT_PUBLIC_SANITY_PROJECT_ID: ${{secrets.STAGING_SANITY_PROJECT_ID}}
-  NEXT_PUBLIC_SANITY_DATASET: staging
+  NEXT_PUBLIC_SANITY_DATASET: production
```

Comment:
This is ok because we want to be able to test changes before rolling out to prod